tests/spark/rflx-builtin_types_tests.adb

```diff
@@ -995,125 +1078,125 @@ package body RFLX.Builtin_Types_Tests is
       Assert (Buffer.all, (250, 85, 85, 85, 85, 85, 85, 85, 175),
               "Invalid insertion in filled buffer with offset 3");
       Buffer.all := (255, 255, 255, 255, 255, 255, 255, 255, 255);
-      RFLX_Types.Insert (RFLX.RFLX_Types.U64'(12297829382473034410), Buffer,
-                         Buffer'First, Buffer'Last, Dynamic_Offset (4), 64, RFLX_Types.Low_Order_First);
+      RFLX_Types.Operations.Insert (RFLX.RFLX_Types.U64'(12297829382473034410), Buffer,
+                                    Buffer'First, Buffer'Last, Dynamic_Offset (4), 64, RFLX_Types.Low_Order_First);
       Assert (Buffer.all, (250, 170, 170, 170, 170, 170, 170, 170, 175),
               "Invalid insertion in filled buffer with offset 4");
       Buffer.all := (255, 255, 255, 255, 255, 255, 255, 255, 255);
-      RFLX_Types.Insert (RFLX.RFLX_Types.U64'(12297829382473034410), Buffer,
-                         Buffer'First, Buffer'Last, Dynamic_Offset (5), 64, RFLX_Types.Low_Order_First);
+      RFLX_Types.Operations.Insert (RFLX.RFLX_Types.U64'(12297829382473034410), Buffer,
+                                    Buffer'First, Buffer'Last, Dynamic_Offset (5), 64, RFLX_Types.Low_Order_First);
       Assert (Buffer.all, (234, 85, 85, 85, 85, 85, 85, 85, 191),
               "Invalid insertion in filled buffer with offset 5");
       Buffer.all := (255, 255, 255, 255, 255, 255, 255, 255, 255);
-      RFLX_Types.Insert (RFLX.RFLX_Types.U64'(12297829382473034410), Buffer,
-                         Buffer'First, Buffer'Last, Dynamic_Offset (6), 64, RFLX_Types.Low_Order_First);
+      RFLX_Types.Operations.Insert (RFLX.RFLX_Types.U64'(12297829382473034410), Buffer,
+                                    Buffer'First, Buffer'Last, Dynamic_Offset (6), 64, RFLX_Types.Low_Order_First);
       Assert (Buffer.all, (234, 170, 170, 170, 170, 170, 170, 170, 191),
               "Invalid insertion in filled buffer with offset 6");
       Buffer.all := (255, 255, 255, 255, 255, 255, 255, 255, 255);
-      RFLX_Types.Insert (RFLX.RFLX_Types.U64'(12297829382473034410), Buffer,
-                         Buffer'First, Buffer'Last, Dynamic_Offset (7), 64, RFLX_Types.Low_Order_First);
+      RFLX_Types.Operations.Insert (RFLX.RFLX_Types.U64'(12297829382473034410), Buffer,
+                                    Buffer'First, Buffer'Last, Dynamic_Offset (7), 64, RFLX_Types.Low_Order_First);
       Assert (Buffer.all, (170, 85, 85, 85, 85, 85, 85, 85, 255),
               "Invalid insertion in filled buffer with offset 7");
 
       Buffer.all := (0, 0, 0, 0, 0, 0, 0, 0, 0);
-      RFLX_Types.Insert (RFLX.RFLX_Types.U64'(4294967295), Buffer, Buffer'First, Buffer'Last, Dynamic_Offset (0), 64,
-                         RFLX_Types.Low_Order_First);
+      RFLX_Types.Operations.Insert (RFLX.RFLX_Types.U64'(4294967295), Buffer, Buffer'First, Buffer'Last, Dynamic_Offset
+                                    (0), 64, RFLX_Types.Low_Order_First);
```

Comment:
Strange position for a line break. Wouldn't it be better to move `Dynamic_Offset` also to the new line?

Reply:
Fixed.